src/bin/ch-remote.rs

```diff
@@ -184,9 +186,16 @@ fn resize_api_command(
         None
     };
 
+    let desired_balloon: Option<u64> = if let Some(balloon) = balloon {
```

Comment:
Sorry to be picky with the naming here, but `desired_balloon` sounds like the user is providing the size for the balloon, while actually it means the expected size of the RAM, by inflating the balloon.
So I suggest this variable should be called `desired_ram_w_balloon`. Please update everywhere this can apply.

Reply:
Fixed.